src/app/custom/translations.tsv

```diff
@@ -533,5 +534,6 @@
 "import_file_text"	"Drop your file on this page"	"Glisser un fichier sur cette page"
 "new_subresource"	"New subresource"	"Nouvelle sous-ressource"
 "delete"	"Delete"	"Supprimer"
-"confirm_delete_subresource"	"This subresource will be definitively deleted, Are you sure?"	"Cette sous-resource va être supprimée. Êtes-vous sûr de vouloir continuer ?"
+"confirm_delete_subresource"	"This subresource and all its fields will be definitively deleted, Are you sure?"	"Cette sous-resource ainsi que tous les champs associés vont êtres supprimées. Êtes-vous sûr de vouloir continuer ?"
```

Comment:
I think it's "sous-ressource" in french with two "s"
and "supprimées" should be "supprimés"